Tribler/Test/Community/Multichain/statistics/test_page_rank.py

```diff
@@ -1,3 +1,6 @@
+"""
+Tests for the incremental PageRank.
+"""
 from unittest import TestCase
 
 from networkx import gnm_random_graph
```

Comment:
Since we're not using networkx anymore, do you think it's a good thing to keep this dependency only for testing purposes?